data/transactions/logic/assembler_test.go

```diff
@@ -536,7 +541,7 @@ func TestAssemble(t *testing.T) {
 	}
 }
 
-var experiments = []uint64{pairingVersion, spliceVersion}
+var experiments []uint64
```

Comment:
Minor quibble: I'd prefer it keeps the same form, even while empty.  People are more likely to update it.
```suggestion
var experiments []uint64 = []uint64{}
```
unless some linter complains.